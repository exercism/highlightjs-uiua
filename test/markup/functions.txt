Cube ← ××..
Cube = (××..)
TimesThree ← |1.1 ×3
TimesThree ← |1 ×3
