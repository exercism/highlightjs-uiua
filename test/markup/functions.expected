<span class="hljs-title function_ invoke__">Cube</span> <span class="hljs-built_in">←</span> <span class="hljs-built_in">×</span><span class="hljs-built_in">×</span><span class="hljs-built_in">.</span><span class="hljs-built_in">.</span>
<span class="hljs-title function_ invoke__">Cube</span> <span class="hljs-built_in">=</span> <span class="hljs-punctuation">(</span><span class="hljs-built_in">×</span><span class="hljs-built_in">×</span><span class="hljs-built_in">.</span><span class="hljs-built_in">.</span><span class="hljs-punctuation">)</span>
